README.md

```diff
@@ -1,88 +1,112 @@
+We have the following [branches](https://github.com/bndtools/bnd/branches/all):
 
-[CodeQL](https://github.com/bndtools/bnd/security/code-scanning?query=tool%3ACodeQL) is used for continuous security analysis.
+* `master` – Where the work for the next release takes place. Pushes on this branch release to the snapshot repository
+* `classic` – A continuation of 6.4 compiled on Java 1.8 for the poor developers stuck in the limbo of 1.8 and 17.
+* `next` – Release branch to Maven central. Is used during the [release process](https://github.com/bndtools/bnd/wiki/Release-Process).
 
-A Maven snapshot repository of the latest build is available at <https://bndtools.jfrog.io/bndtools/libs-snapshot-local>.
+We generally do not use other branches. Issues and features are handled through PR's from forked repos.
 
-### Using the latest development SNAPSHOT build of Bnd/Bndtools
+### Issues
 
-* [Bnd Gradle Plugins](gradle-plugins/README.md#using-the-latest-development-snapshot-build-of-the-bnd-gradle-plugins)
-* [Bnd Maven Plugins](maven/README.md#using-the-latest-development-snapshot-build-of-the-bnd-maven-plugins)
-* Bndtools snapshot p2 update site: <https://bndtools.jfrog.io/bndtools/update-snapshot>
+[Open issues](https://github.com/bndtools/bnd/issues) are the one _actively scheduled_ to go into the next milestone(s). 
 
-### Using the latest development Milestone/Release Candidate build of Bnd/Bndtools
+_All other issues will be closed_. However, some issues we close because we just lack people that are interested in working on them. We explicitly mark those issues with [`abeyance`](https://github.com/bndtools/bnd/issues?q=is%3Aissue+label%3Aabeyance+). If you got time to spare, do not hesitate to open them and indicate you're willing to work on them. We will then properly schedule and track them. If you need to find them, use this filter: `is:issue label:abeyance`, make sure `is:open` is not in the filter.
 
-* [Bnd Gradle Plugins](gradle-plugins/README.md#using-the-latest-milestonerelease-candidate-build-of-the-bnd-gradle-plugins)
-* [Bnd Maven Plugins](maven/README.md#using-the-latest-milestonerelease-candidate-build-of-the-bnd-maven-plugins)
-* Bndtools Milestone/Release Candidate p2 update site: <https://bndtools.jfrog.io/bndtools/update-rc>
+### Milestones
 
-![Powered by Artifactory](https://github.com/bndtools/bnd/raw/master/docs/img/Powered-by-artifactory_04.png)
+We try to have a release every 3-6 months and plan these with [milestones](https://github.com/bndtools/bnd/milestones).
 
-## Release
+### PRs
 
-Release versions of Bnd artifacts including Maven and Gradle plugins are available from [Maven Central](https://search.maven.org/search?q=g:biz.aQute.bnd). Release versions of the Bnd Gradle plugins are also available from the [Gradle Plugin repository](https://plugins.gradle.org/search?term=biz.aQute.bnd).
+[PRs](https://github.com/bndtools/bnd/pulls) should be submitted via another repo. We are using the following branch names:
```

Comment:
I don't think there is much use in attempting to prescribe the branch names people use in their own forks. The main thing is that they don't make a PR from master. You may also want to make clear that all PRs must be against master. Changes to classic are a special case and perhaps left to the maintainers alone.